dora/underfs/obs/src/main/java/alluxio/underfs/obs/OBSUnderFileSystem.java

```diff
@@ -368,6 +370,18 @@ protected ObjectStatus getObjectStatus(String key) {
     }
   }
 
+  @Override
+  public UfsStatus getStatus(String path) throws IOException {
+    ObjectStatus status = getObjectStatus(stripPrefixIfPresent(path));
+    if (!isDirectory(path)) {
```

Comment:
you can check if it is the directory or file first, and then getObjectStatus. because if it is a directory, it doesn't need to getObjectStatus.

Reply:
ok,I'll change it

Reply:
> you can check if it is the directory or file first, and then getObjectStatus. because if it is a directory, it doesn't need to getObjectStatus.

I would like to ask a question，Java 11 Integration Tests / modules: (alluxio.client.cli.fs.**) (pull_request)  check failed，I can't find the problem，how to find the problem?

Reply:
> you can check if it is the directory or file first, and then getObjectStatus. because if it is a directory, it doesn't need to getObjectStatus.

I have already made changes

Reply:
Don't worry, I will rerun the unit test and check if it's your fault.

Reply:
It's probably a flaky test case